src/pages/home/ReportScreen.js

```diff
@@ -142,7 +142,6 @@ class ReportScreen extends React.Component {
     }
 
     componentWillUnmount() {
-        clearTimeout(this.loadingTimerId);
```

Comment:
Was this just old code that should have been removed in a previous PR?

Reply:
hmm, this might have been related to the PR that was merged in, but this was not removed???
from what I remember it was to remove the loader after a certain time, but we don't use it anymore.
I know I removed the timer in this [commit](https://github.com/Expensify/App/pull/10164/commits/b301b28689ccbd4dd7fb3073ad03927e4b599443)